src/Item.php

```diff
@@ -114,6 +114,14 @@ public function hasAttributes(): bool
         return !empty($this->toArray());
     }
 
+    /**
+     * @return Meta
+     */
+    public function getDataMeta(): Meta
```

Comment:
I don't really know how else to access `meta` or what to call this function, but I do think it's a bit confusing. Open to better suggestions!

Reply:
Just pushed a commit which removes this.